ticktok/http_session.py

```diff
@@ -0,0 +1,22 @@
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+
+
+def create_retry_session(
+        retries=6,
```

Comment:
shoudn't values defined in conf file?

Reply:
There's no conf for a library :) I need to delete this file actually as I don't use it currently